force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -10906,6 +10906,13 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Notify user component has been deprecated.</shortDescription>
         <value>This component has been deprecated.</value>
     </labels>
+    <labels>
+        <fullName>lblEmpty</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label to display Empty.</shortDescription>
+        <value>empty</value>
+    </labels>
```

Comment:
@sam-knox Adding a new custom label for Contact Merge page. The label is used when there is no value on a specific field in contact merge(align with core Contact Merge)
![Screen Shot 2021-11-09 at 4 12 57 PM](https://user-images.githubusercontent.com/57271120/141178198-1a0a47f1-95f3-415d-84ec-01dcfcefe46a.png)
.